src/AdminBarExtension.php

```diff
@@ -6,20 +6,42 @@
 
 
 use Baraja\AdminBar\Panel\BasicPanel;
+use Baraja\Localization\Localization;
 use Nette\DI\CompilerExtension;
 use Nette\DI\Definitions\ServiceDefinition;
 use Nette\PhpGenerator\ClassType;
+use Nette\Schema\Elements\Structure;
+use Nette\Schema\Elements\Type;
+use Nette\Schema\Schema;
 use Nette\Security\User;
+use stdClass;
 
 final class AdminBarExtension extends CompilerExtension
 {
+	public function getConfigSchema(): Schema
+	{
+		return new Structure(
+			['defaultLocale' => (new Type('string'))->default('en')],
+		);
+	}
+
+
 	public function beforeCompile(): void
 	{
+		/** @var stdClass{defaultLocale: string} $config */
+		$config = $this->config;
+
 		$builder = $this->getContainerBuilder();
 
-		$builder->addDefinition($this->prefix('basicPanel'))
-			->setFactory(BasicPanel::class)
+		$basicPanel = $builder->addDefinition($this->prefix('basicPanel'));
+		$basicPanel->setFactory(BasicPanel::class)
+			->addSetup('setDefaultLocale', [$config->defaultLocale])
 			->setAutowired(BasicPanel::class);
+
+		if (class_exists(Localization::class) && is_string($builder->getByType(Localization::class)) !== false) {
+			$localization = $builder->getDefinitionByType(Localization::class);
+			$basicPanel->addSetup('setLocalization', [$localization]);
+		}
```

Comment:
Odeber. Inject závislosti řeší DIC automaticky.

Reply:
Vytvořeno kvůli chybě ve analýze v BasicPanel.php - Avoid unused parameters on PHP 8: Constructor property

Reply:
Funkčnost je důležitější, než statická analýza. Lepší je mít všechny závislosti přiznané v konstruktoru, i když jsou nepovinné.